documentation-website/cypress/e2e/quick-start.cy.ts

```diff
@@ -11,12 +11,12 @@ describe('usage/quick-start', () => {
     cy.url().should('eq', BASE_URL + page.quickStart,);
   },);
   it('title matches', () => {
-    cy.title().should('eq', 'Quick Start | @idrinth/api-bench',);
+    cy.title().should('eq', 'Quick Start | @idrinth-api-bench/api-bench',);
```

Comment:
this package will never exist, please revert the change for now

Reply:
Reverted